config/config.go

```diff
@@ -38,3 +42,7 @@ func GetReleaseString() string {
 
 	return fmt.Sprintf("Owncast v%s-%s (%s)", versionNumber, buildPlatform, gitCommit)
 }
+
+func GetTranscoderLogFilePath() string {
```

Comment:
Sorry, one nit!  Could you remove Transcoder from this since it can be used for all logs, not just a single log?  `GetLogPath` or `GetLogDirectory` or something that makes sense.

Reply:
My idea was that LogDirectory would be shared for all logs, and then we have a function for each of the other logs but those use the LogDirectory variable.

eg:
If you wanted to add another type of log, you'd add the function like so:

```go
func GetMyOtherLogFilePath() string {
    return filepath.Join(LogDirectory, "my_other_log.log")
}
```

and then you can use that for the other logs. This way we avoid having a bunch of filepath.Join(LogDirectory, "my_other_log.log") scattered everywhere in the code base, you just need to edit that one function.

What do you think?

Reply:
I get it, that works for me!  Thanks for clarifying!